src/components/movie-view/movie-view.jsx

```diff
@@ -1,15 +1,65 @@
 import React from "react";
+// not using useEffect... yet
+import { useEffect, useState } from 'react';
 import { useParams } from "react-router";
 import { Link } from "react-router-dom";
+import { Tooltip } from 'react-tooltip'
 import "./movie-view.scss";
 
+//note to self: fix class names
+
+// This is the "single movie view" that appears when you click a movie
+
 export const MovieView = ({ movies }) => {
   const { movieId } = useParams();
 
   const movie = movies.find((m) => m.id === movieId);
 
+  const [isFavorite, setToFavorite] = useState(false);
+
+  console.log(movie.ImagePath);
+
+  const handleAddToFavorites = async (movieId) => {
+    // event.preventDefault();
+
+    //add in "try {}"?
+
+    const user = JSON.parse(localStorage.getItem('user'));
+    const token = localStorage.getItem('token');
+
+    const response = await fetch(`https://movie-db-fullstack-2-27a48700ab77.herokuapp.com/users/${user.Username}/movies/${movieId}`, {
+      method: "POST",
```

Comment:
should be PUT here

https://github.com/ArthurShelter/full-stack-2/blob/main/index.js#L243